ynr/apps/moderation_queue/templates/moderation_queue/includes/inline-lock-table-view.html

```diff
@@ -0,0 +1,48 @@
+<tr>
+  <td>
+    Suggested by<br>
```

Comment:
This is functional, but if feels a bit randomly sneezed onto the page. We can probably clean this up a bit at some point

![Screenshot at 2019-04-04 16-03-46](https://user-images.githubusercontent.com/6025893/55566337-626f1700-56f3-11e9-9da6-a35943f69b83.png)